### DISCUSSION TITLE
Update to latest version of BOB v2.4.9

### DISCUSSION THREAD

**Discussion by devinhh** [IBM/ibmi-tobi#206] [Q&A] [answered]
How do I get the latest version. When I use yum install bob, it comes back with the message: "Package bob-2.9.1-1.ppc64 already installed and lastest version Nothing to do"

**Comment by devinhh** [comment]
I meant to say "Package bob-2.3.9-1.ppc64

**Comment by beckhamrryyaann** [comment]
I had about the same issue when I first started using Bob.  There are 2
issues and discussions I opened up with the Bob dev team.  Trying searching
for them on this Bob GitHub page.

**Reply by beckhamrryyaann** [reply]
Look at issue 124
https://github.com/IBM/ibmi-bob/issues/124

also look at this discussion.  It may prove helpful.  
https://github.com/IBM/ibmi-bob/discussions/150

**Reply by devinhh** [reply, marked as answer]
Thanks for your help. After updating YUM I was able to update BOB.
